modules/settings.js

```diff
@@ -31,7 +31,18 @@ export const registerModuleSettings = async function() {
         default: false,
         config: true,
         onChange: async s => {}
-    }),
+    })
```

Comment:
Something weird going on with the indentation here. Please ensure it follows the same format as other settings.